crates/motsu/src/shims.rs

```diff
@@ -189,7 +189,7 @@ unsafe extern "C" fn emit_log(_: *const u8, _: usize, _: usize) {
 ///
 /// # Panics
 ///
-/// May panic if fails to parse `ACCOUNT_CODEHASH` as a keccack hash.
+/// * If fails to parse `ACCOUNT_CODEHASH` as a keccack hash.
```

Comment:
```suggestion
/// * If it fails to parse the account code hash as a keccak hash.
```
- there is no `ACCOUNT_CODEHASH`.
- is this even a correct panic description given the implementation?

Reply:
replied [here](https://github.com/OpenZeppelin/stylus-test-helpers/pull/34#discussion_r1967483970)